### PR TITLE
ones_like and zeros_like dtype parameter is documented

### PR DESCRIPTION
**Pull request by shabanian**
fix gh-4548
@adri-romsor
@lamblin

### CODE REVIEW

File: doc/library/tensor/basic.txt [hunk 1]
```diff
@@ -664,17 +664,22 @@ Creating Tensor
...
 
-    :param x: tensor that has same shape as output
+    :param x: tensor that has the same shape as output
+    :param dtype: data-type, optional
 
-    Returns a tensor filled with 0s that has same shape as `x`.
+    Returns a tensor the shape of x filled with zeros of the type of dtype.
+    
 
 .. function:: ones_like(x)
```
Comment (nouiz) [inline doc/library/tensor/basic.txt]: add the dtype param here too.

File: doc/library/tensor/basic.txt [hunk 2]
```diff
@@ -664,17 +664,22 @@ Creating Tensor
 ===============
 
 
-.. function:: zeros_like(x)
+.. function:: zeros_like(x, dtype=None)
 
-    :param x: tensor that has same shape as output
+    :param x: tensor that has the same shape as output
+    :param dtype: data-type, optional
```
Comment (nouiz) [inline doc/library/tensor/basic.txt]: Tell that by default, it will be x.dtype.

**Comment by nouiz** [comment]
For me it look good. @adri-romsor do you see something else?

**Comment by adri-romsor** [comment]
Looks good to me too :)

File: doc/library/tensor/basic.txt [hunk 3]
```diff
@@ -677,7 +678,8 @@ Creating Tensor
   
     :param x: tensor that has the same shape as output
     :param dtype: data-type, optional
-  
+                  By default, it will be x.dtype.
```
Comment (nouiz) [inline doc/library/tensor/basic.txt]: I merged too fast. You forgot to add the dtype param on line 676

**Pull request merged**